azurerm/resource_arm_log_analytics_linked_service.go

```diff
@@ -56,10 +56,13 @@ func resourceArmLogAnalyticsLinkedService() *schema.Resource {
 			},
 
 			"linked_service_properties": {
-				Type:     schema.TypeList,
-				Optional: true,
-				Computed: true,
-				ForceNew: true,
+				Type:          schema.TypeList,
+				Optional:      true,
+				Computed:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"resource_id"},
+				MaxItems:      1,
+				Deprecated:    "This property has been deprecated in favour of the 'resource_id' property and will be removed in version 2.0 of the provider",
```

Comment:
can we add this [to the 2.0 upgrade guide](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/guides/2.0-upgrade-guide.html.markdown) too?